controllers/event.controller.js

```diff
@@ -19,6 +20,7 @@ exports.create = async (req, res) => {
   event
     .save()
     .then((data) => {
+      rewardPointsController.addRewardPoints(req.user.username, 50, `For creating a new event`);
```

Comment:
make the reward points constants at the beginning of the file so that its easier to find out and edit